manifests/j/JetBrains/ReSharper/EarlyAccess/2021.2.EAP5/JetBrains.ReSharper.EarlyAccess.yaml

```diff
@@ -0,0 +1,29 @@
+PackageIdentifier: JetBrains.ReSharper.EarlyAccess
+PackageVersion: 2021.2.EAP5
```

Comment:
```suggestion
PackageVersion: 2021.2 EAP 5
```
Please update the version folder as well.

Reply:
I think it will be 2021.2.EAP5, but if you think <SPACE> will be there, then do tell me.
![image](https://user-images.githubusercontent.com/83997633/124785717-b5844400-df64-11eb-8db6-992a051b6a64.png)

Reply:
Please reply... 🙏🏻

Reply:
Version needs to match Control Panel otherwise it'll end up in an upgrade loop with `winget upgrade`

Reply:
Okay

Reply:
Doing the changes